x-pack/plugins/data_visualizer/public/application/common/components/link_card/link_card.tsx

```diff
@@ -62,10 +62,10 @@ export const LinkCard: FC<LinkCardProps> = ({
         color="subdued"
         {...linkHrefAndOnClickProps}
       >
-        <EuiFlexGroup gutterSize="l" responsive={true}>
-          <EuiFlexItem grow={false} style={{ paddingTop: '8px' }}>
+        <EuiFlexGroup gutterSize="s" responsive={true}>
+          <EuiFlexItem grow={false} style={{ paddingTop: '4px' }}>
```

Comment:
Please use variables from the `useEuiTheme` hook instead of the static number

Reply:
Updated here [`2d35742` (#147137)](https://github.com/elastic/kibana/pull/147137/commits/2d35742e343066904b17034fa60a0b1c3fb5536e)